DMCompiler/Compiler/DM/DMParser.cs

```diff
@@ -1889,9 +1889,12 @@ public DMASTProcStatementLabel Label(DMASTIdentifier expression)
         }
 
         public DMASTExpression ExpressionPrimary(bool allowParentheses = true) {
-            if (allowParentheses && Check(TokenType.DM_LeftParenthesis)) {
+            if (allowParentheses && Check(TokenType.DM_LeftParenthesis))
+            {
                 Whitespace();
                 DMASTExpression inner = Expression();
+                Newline();
+                Whitespace(includeIndentation: true);
```

Comment:
`BracketWhitespace()` is meant to be used for whitespace inside brackets/parentheses which allows delimiters. The first `Whitespace()` here should be changed too.
```suggestion
                BracketWhitespace();
                DMASTExpression inner = Expression();
                BracketWhitespace();
```